package.json

```diff
@@ -44,16 +47,10 @@
     "pre-commit": "^1.2.0",
     "prettier": "^2.0.5",
     "rimraf": "^2.6.0",
-    "rollup": "^0.41.6",
-    "rollup-plugin-buble": "^0.15.0",
-    "rollup-plugin-commonjs": "^8.4.1",
-    "rollup-plugin-node-resolve": "^3.3.0",
-    "rollup-plugin-replace": "^1.1.0",
-    "uglify-js": "^3.4.7",
+    "rollup": "^2.23.0",
+    "uglify-js": "^3.10.1",
     "uppercamelcase": "^3.0.0",
-    "vue": "^2.5.17",
-    "whatwg-fetch": "^3.0.0",
-    "yarn-or-npm": "^2.0.0"
```

Comment:
It seems we're pretty bought in with `npm`, and we don't have a `yarn.lock`, so I removed this. Not opposed to switching to Yarn, but I feel like it's less than ideal to support both.